### DISCUSSION TITLE
Forward Slashes in ID3 TXXX tags

### DISCUSSION THREAD

**Discussion by imposterdroids** [beetbox/beets#5241] [Q&A] [answered]
Forgive me, as I'm really new to beets.  This has probably been addressed somewhere before, but i've been unable to find the answer.

I have some existing Musicbrainz ID3 tags on mp3 files. For example, a TXXX frame for the musicbrainz album type, and it contains the text “album/compilation”. I’m finding that when imported, beets rewrites these tags with the text: “[/‘/a/l/b/u/m/’/,/ /‘/c/o/m/p/i/l/a/t/i/o/n/’/]”

Can someone tell me why? Is this a separator issue? Does beets not handle the forward slash correctly?  These are ID3v2.3 tags btw.   Thanks for any help.

**Comment by pyyhttu** [comment, marked as answer]
This should [answer](https://github.com/beetbox/beets/discussions/5183#discussioncomment-9057026).

**Reply by imposterdroids** [reply]
Appreciate it pyyhttu.
